### DISCUSSION TITLE
Help request: diagnosing issue

### DISCUSSION THREAD

**Discussion by bcourtney5965** [vercel/next.js#20105] [Help]
I ran into several issues.  Everytime I tried to fix one it seemed to only make it worse.  I wanted to install a NextJS starter app.  Now I don't have access to the npm command.  I'm thinking about uninstalling node, npm, and nvm and reinstalling with brew.  I'm trying to figure out what  I did wrong here... Any advice is much appreciated.

1.) I was trying to install a nextJS starter app so I ran this command unsuccessfully a few times:
npx create-next-app next-app-test

upon reading the error logs they said
7 silly fetchPackageMetaData error for create-next-app@latest Response timeout while trying to fetch https://registry.npmjs.org/create-next-app (over 30000ms)

11 verbose stack FetchError: Response timeout while trying to fetch https://registry.npmjs.org/create-next-app (over 30000ms)

Googling the error showed this: https://github.com/npm/cli/issues/1185

I realized I was using this "bad" npm version: 6.14.4

2.) So I ran npm install -g npm@latest

It said I was missing write access to a ton of things:

3.) So I ran sudo npm install -g npm@latest and it errored saying that a file already existed:

/usr/local/share/man/man1/npm-access.1

and to remove it OR do it unsafely.  So I removed it

4. rm /usr/local/share/man/man1/npm-access.1

5. npm install -g npm@latest
but it says: command not found: npm

which npm also shows: command not found

node version: v12.16.3
npm version: was 6.14.4 
https://github.com/npm/cli/issues/1185

I wanted to install a NextJS starter app.  Now I don't have access to the npm command.  Like I mentioned at the top I'm thinking about uninstalling node, npm, and nvm and reinstalling with brew.  Any advice?

**Comment by michrome** [comment]
If you run `nvm --version` and get a version number back, then your best bet is to reinstall Node.js which will give you npm. Something like:

```bash
mike@MacBook-Pro ~ % nvm --version
0.35.3

mike@MacBook-Pro ~ % nvm install --lts
Installing latest LTS version.
Downloading and installing node v12.18.0...
Downloading https://nodejs.org/dist/v12.18.0/node-v12.18.0-darwin-x64.tar.gz...
############################################################################################################################## 100.0%
Computing checksum with shasum -a 256
Checksums matched!
Now using node v12.18.0 (npm v6.14.4)
Creating default alias: default -> lts/* (-> v12.18.0)

mike@MacBook-Pro ~ % node --version
v12.18.0
```

You can then install Next via npm from there.
